grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy

```diff
@@ -337,7 +337,7 @@ class InvoiceService {
         InvoiceItem invoiceItem = new InvoiceItem(
             budgetCode: orderAdjustment.budgetCode,
             product: orderAdjustment.orderItem?.product,
-            glAccount: orderAdjustment.glAccount ?: orderAdjustment.orderItem?.glAccount,
+            glAccount: orderAdjustment.orderItem?.glAccount ?: orderAdjustment.orderAdjustmentType?.glAccount,
```

Comment:
This doesn't feel right. It seems the order of precedence should be 

order adjustment (override) -> order item (override) -> order adjustment type (default)

Reply:
@jmiranda You're right, I've missed that. I updated PR.

Reply:
@awalkowiak @DominikaHatala If the ticket says to do it the way you did it, then ignore my comments and merge it. If it's not clear then go with the precedence order that I specified.

```
orderAdjustment?.glAccount ?: orderAdjustment?.orderItem?.glAccount ?: orderAdjustment?.orderAdjustmentType?.glAccount

```

Reply:
It was not clearly specified in the ticket so I applied your suggestions.